app/models/work-item.ts

```diff
@@ -399,20 +399,27 @@ export async function getWorkItemsByJobIdAndStepIndex(
  * @param notUpdatedForMinutes - jobs with updateAt older than notUpdatedForMinutes ago
  * will be joined with the returned work items
  * @param jobStatus - only jobs with this status will be joined
+ * @param startingId - the work item id to begin the query with, i.e. query work items with id greater than startingId
+ * @param batchSize - the batch size
  * @returns - all work item ids associated with the jobs that
  * met the updatedAt and status constraints
  */
 export async function getWorkItemIdsByJobUpdateAgeAndStatus(
   tx: Transaction,
   notUpdatedForMinutes: number,
   jobStatus: JobStatus[],
+  startingId = 0,
```

Comment:
It's slightly confusing that you don't actually include `startingId` in the results. I get the logic, just find the naming a little confusing, although I can't think of a better alternative.

Reply:
yeah, we had similar discussion on this on `work-failer`. Sorry that I have not found a better alternative and I think it is good to use the same naming convention as the existing code in `work-failer`.